src/lib/accessibility/Behaviors/Chat/chatBehavior.ts

```diff
@@ -34,4 +34,13 @@ const ChatBehavior: Accessibility = (props: any) => ({
   },
 })
 
+const getLastTabbableElement = (root: HTMLElement): HTMLElement => {
+  const chatItemsElements = root.querySelectorAll(
+    `[${CHAT_FOCUSZONE_ATTRIBUTE}] .ui-chat__item > [${IS_FOCUSABLE_ATTRIBUTE}]`,
+  )
+  return (chatItemsElements.length > 0
```

Comment:
maybe 
`return chatItemsElements.length > 0 ? (chatItemsElements[chatItemsElements.length - 1] as HTMLElement) : null`

Reply:
yeah-yeah, fixed together with the above comment. Thank you

Reply:
yes, this would be better - the reason I've suggested to enclose everything in braces was to guarantee to preserve type for the evaluated expression. This moment will become critical in case if `strictNullCheck` flag will be enabled for TS compiler. At the same time, this type is explicitly defined by function return type, and, thus, we don't need to brace everything, thanks!